src/config.rs

```diff
@@ -1742,6 +1743,23 @@ mod tests {
         // Sanity check that key bindings are being parsed
         assert!(!config.key_bindings.is_empty());
     }
+
+    #[test]
+    fn update_dynamic_title() {
+        let config: Config = ::serde_yaml::from_str(ALACRITTY_YML)
+            .expect("deserialize config");
+        assert!(config.dynamic_title);
+
+        let mut options = Options::default();
+        let config = config.update_dynamic_title(&options);
+        assert!(config.dynamic_title);
```

Comment:
Testing the default here makes a little more sense, since it's testing that by default nothing is overridden, however I think if you want to test that it should be a separate test.
However it should just test that the config option is not changed, instead of testing that dynamic title is enabled.